src/exampleWorkerScript.worker.ts

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+export { };
```

Comment:
I only added this to make typescript happy, but I'm not sure if it's the right approach or not. I have to test to see if this is actually working, we don't really have tests for workers explicitly (but we should!)